<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>以太网封帧器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_eth">
       <attribute name="title">
        <string>以太网帧</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_7">
             <property name="text">
              <string>E6:1A:26:AE:F2:0C</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_6">
             <property name="text">
              <string>A2:4B:7B:05:B0:44</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>矢MAC</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>源MAC</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>使用虚拟信道中商定的MAC地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>接收的以太网帧</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>解析 &gt;&gt;&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_ipv4">
       <attribute name="title">
        <string>IPV4帧</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>分片偏移</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEdit_10">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>192.168.254.2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="lineEdit_11">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>CS0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF21</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF22</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF23</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF31</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF33</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF41</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF42</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF43</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EF PHB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>VOICE-ADMIT</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>DSCP</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QPlainTextEdit" name="plainTextEdit_4"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>ECN</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>内容</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>标识</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>192.168.254.1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>源IP</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>TTL</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="lineEdit_12">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="lineEdit_9">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>64</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="comboBox_2">
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>01</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>00</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>标志</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>矢IP</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>使用虚拟信道中商定的IP地址</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_ipv6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>IPV6帧</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_v6">
        <item row="7" column="2">
         <widget class="QLineEdit" name="lineEdit_8_v6">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>2409:8a1e:76e3:3250:a584:113b:1fe2:33d7</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="lineEdit_11_v6">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_16_v6">
          <property name="text">
           <string>下一个头部</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_13_v6">
          <property name="text">
           <string>内容</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2_v6">
          <property name="text">
           <string>ECN</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEdit_10_6">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="lineEdit_3_v6">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>2409:8a1e:76e3:3250:1cf0:850e:4e32:7a6a</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_14_v6">
          <property name="text">
           <string>跳数限制</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_15_v6">
          <property name="text">
           <string>流标签</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_12_v6">
          <property name="text">
           <string>矢IP</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="comboBox_v6">
          <item>
           <property name="text">
            <string>CS0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CS7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF21</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF22</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF23</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF31</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF33</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF41</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF42</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF43</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EF PHB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>VOICE-ADMIT</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QPushButton" name="pushButton_3_v6">
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="comboBox_2_v6">
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>01</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>00</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_11_v6">
          <property name="text">
           <string>源IP</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_v6">
          <property name="text">
           <string>DSCP</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="lineEdit_9_v6">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>64</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QPlainTextEdit" name="plainTextEdit_4_v6"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_arp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>ARP</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>暂时没有写ARP包生成器</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>解析</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>帧内容</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>源MAC</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>矢MAC</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>控制信息</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="plainText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>载荷</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit_2">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>原始PDU</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit_3">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>连接</string>
    </property>
    <addaction name="actionTCP"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actiondakai">
   <property name="text">
    <string>dakai</string>
   </property>
  </action>
  <action name="actionTCP">
   <property name="text">
    <string>设置信道</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
